justfile

```diff
@@ -151,11 +151,11 @@ web-dev-up:
         ln -s "./assets/xaynia_v0002" "./web-api/assets"
     fi
     export HOST_PORT_SCOPE=30
-    docker-compose -p "$PROJECT" -f "./web-api/compose.db.yml" up --detach --remove-orphans --build
+    podman-compose -p "$PROJECT" -f "./web-api/compose.db.yml" up --detach --remove-orphans --build
```

Comment:
:wrench:  We decided to always use docker due to the huge amount of problems we had caused by subtle differences between both of them. I think we might also be using some options podman doesn't support and silently ignores.

Reply:
Done